tests/deploy_test.go

```diff
@@ -155,6 +200,69 @@ func Test_ListNamespaces(t *testing.T) {
 	}
 }
 
+func compareDeployAndStatus(deploy types.FunctionDeployment, status types.FunctionStatus) error {
+	if deploy.Service != status.Name {
+		return fmt.Errorf("got %v, expected name %s", status.Name, deploy.Service)
+	}
+	if deploy.Image != status.Image {
+		return fmt.Errorf("got %v, expected image %s", status.Image, deploy.Image)
+	}
+	if deploy.Namespace != status.Namespace {
+		return fmt.Errorf("got %v, expected Namespace %s", status.Namespace, deploy.Namespace)
+	}
+	if deploy.EnvProcess != status.EnvProcess {
+		return fmt.Errorf("got %v, expected EnvProcess %s", status.EnvProcess, deploy.EnvProcess)
+	}
+
+	if deploy.ReadOnlyRootFilesystem != status.ReadOnlyRootFilesystem {
+		return fmt.Errorf("got %v, expected ReadOnlyRootFilesystem %v", status.ReadOnlyRootFilesystem, deploy.ReadOnlyRootFilesystem)
+	}
+
+	if !reflect.DeepEqual(deploy.EnvVars, status.EnvVars) {
+		return fmt.Errorf("got %v, expected EnvVars %v", status.EnvVars, deploy.EnvVars)
+	}
+
+	err := strSliceEqual(deploy.Constraints, status.Constraints)
+	if err != nil {
+		return fmt.Errorf("incorrect Constraints: %s", err)
+	}
+
+	err = strSliceEqual(deploy.Secrets, status.Secrets)
+	if err != nil {
+		return fmt.Errorf("incorrect Secrets: %s", err)
+	}
+
+	if !reflect.DeepEqual(deploy.Limits, status.Limits) {
+		return fmt.Errorf("got %v, expected Limits %v", status.Limits, deploy.Limits)
+	}
+
+	if !reflect.DeepEqual(deploy.Requests, status.Requests) {
```

Comment:
When @Waterdrips and I were talking about this, it seemed like faas-netes was not returning this data. So I'd be interested to see what the test says, and whether we're on an up to date version.

I might be missing it, but do you not populate any requests/limits in the input at this time?